etc/version_check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
```

Comment:
This script uses some non-POSIX features so should require bash or remove the non-POSIX features. You could also paste it into shellcheck.net to see those issues.